src/middlewares/basicAuthMiddleware.js

```diff
@@ -35,15 +35,6 @@ const getSiteId = event => {
     siteId = event.platform;
```

Comment:
@jpaylaga  do you think it makes much more sense for us to remove these whole series of checks ? I think your point here is to catch as much as of where to get the site id.

But I think we can insist on just letting them use clientAuthMiddleware which is what you did already. But at the same time add a code there to set the site id such as

```
handler.event.site = clients[0]
```

Even I thought about it and I think we can add more configuration to it. I'll try do a PR for this one, just for the clientAuthMiddlware

Reply:
@jpaylaga I did  a PR to your branch, can you take a look at this as well, those are the ones that I might suggest we have: https://github.com/jpaylaga/lesgo-framework/pull/1/files

Also that includes the removal of the error handling which should be placed on the http instead of our auth middlwares

Reply:
> @jpaylaga do you think it makes much more sense for us to remove these whole series of checks ? We can just leave this last one for `event.platform` I think your point here is to catch as much as of where to get the site id right ? but currently clientAuth just sets `handler.event.platform`
> 
> But I think we can insist on just letting them use clientAuthMiddleware which is what you did already.
> 
> And as a last though about it, I think we can add more configuration to it. I'll try do a PR for this one, just for the clientAuthMiddlware

I think there might be problems if we insist on using the `clientAuthMiddleware` when we already made it optional for `basicAuthMiddleware` in the first place. Given that, there may be situations wherein no `clientAuthMiddleware` is declared but site id is defined anyhow. I think it would be a nice fallback if it detects the different ways site id is declared?